### PR TITLE
Feature/navbar style

### PR DESCRIPTION
**Pull request by omriattiya**
issue #21 
styling the navbars - colors + responsive + better visualize and color matching

### CODE REVIEW

**Review by AGRU13 — APPROVED** [review]
Hey good work I see you have added a lot of things very quickly.
I won't merge it right now as I have to make some changes in it, but I have labelled the PR as hacktoberfest-accepted .

**Comment by AGRU13** [comment]
@omriattiya Since its my first time in hacktoberfest as maintainer, so do let me know if this PR is not counted even after couple of days.

**Comment by omriattiya** [comment]
@AGRU13 as far as I know, the PR needs to be merged and than starts the 14 days window. I would appreciate it if you merge this PR as soon as possible

**Pull request closed without merging**
